### PR TITLE
added RecylerView 

### PR DESCRIPTION
**Pull request by DrBanner97**

### CODE REVIEW

**Review by Pritom14 — CHANGES_REQUESTED** [review]
misc.xml , modules.xml , vcs.xml files cannot be added. Try to add only the modified files like build.gradle, java and  layout files

**Comment by DrBanner97** [comment]
I removed those files and pushed, try again

**Comment by Pritom14** [comment]
Thank you for the PR. Could you please include the screenshot too.
Let other reviewers review it

File: app/src/main/java/com/nitsilchar/hp/passwordStorage/Adapter/PasswordListReyclerView.java [hunk 1]
```diff
@@ -72,6 +82,46 @@ public void onClick(View v) {
                 }
             });
 
+            layout.setOnLongClickListener(new View.OnLongClickListener() {
+                @Override
+                public boolean onLongClick(View v) {
+                    AlertDialog.Builder dialogBuilder=new AlertDialog.Builder(context);
+                    LayoutInflater inflater=(LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);;
```
Comment (Pritom14) [inline app/src/main/java/com/nitsilchar/hp/passwordStorage/Adapter/PasswordListReyclerView.java:89]: Double semicolons

**Comment by DrBanner97** [comment]
device-2017-10-08-101619

File: app/src/main/java/com/nitsilchar/hp/passwordStorage/Adapter/PasswordListReyclerView.java (hunk 1 above)
Comment (DrBanner97) [reply app/src/main/java/com/nitsilchar/hp/passwordStorage/Adapter/PasswordListReyclerView.java:89]: my bad

**Pull request closed without merging**